tests/test_helpscout_start_date.py

```diff
@@ -0,0 +1,109 @@
+from tap_tester import menagerie, connections, runner
+from datetime import timezone
+import datetime
+
+from base import HelpscoutBaseTest
+
+class StartDateTest(HelpscoutBaseTest):
+
+    # Creating variables to store two different start dates
+
+    start_date_1 = ""
+    start_date_2 = ""
+
+    def name(self):
+        return "tap_helpscout_tests_using_shared_token_chaining"
+
+    def test_name(self):
+        print("Start date test for tap-helpscout")
+
+    def test_run(self):
+
+        self.should_fail_fast()
+
+        self.start_date_1 = self.get_properties()
+
+        # setting current timestamp - 1 , to get all the data which was generated today
+        # log in to helpscout and send out emails to customers/ add customers
+
+        self.start_date_2 = datetime.datetime.utcnow()
+        self.start_date_2 = datetime.datetime.strftime(self.start_date_2,"%Y-%m-%dT00:00:00Z")
+        self.start_date_2 = self.timedelta_formatted(self.start_date_2, days = -2)
+
```

Comment:
These dates should be pinned for the time-being else this test will start to fail in the coming days/weeks as the data ages out.